utility_analysis_new/tests/dp_engine_test.py

```diff
@@ -151,20 +153,14 @@ def test_aggregate_error_metrics(self):
         col = list(col)
 
         # Assert
-        # Assert a singleton is returned
-        self.assertEqual(len(col), 1)
-        # Assert there are 2 AggregateErrorMetrics, one for private partition
-        # selection and 1 for count.
-        self.assertEqual(len(col[0]), 2)
-        # Assert count metrics are reasonable.
-        self.assertAlmostEqual(col[0][1].abs_error_expected, -28, delta=1e-2)
-        self.assertAlmostEqual(col[0][1].abs_error_variance, 146.47, delta=1e-2)
-        self.assertAlmostEqual(col[0][1].rel_error_expected,
-                               -0.933333,
-                               delta=1e-5)
-        self.assertAlmostEqual(col[0][1].rel_error_variance,
-                               0.16275,
-                               delta=1e-5)
+        self.assertEqual(len(col), 10)
+        # Assert count metrics are correct.
+        (self.assertTrue(v[1].per_partition_error == -10) for v in col)
```

Comment:
Cool way to check conditions for the array!

please use `[...]` instead of `(...)` 

**Explanation:** `(x for x in ..)` is lazy, it returns lazy terable , while 
  `[x for x in ..]` is eager and returns list